libs/deserializer/src/main/java/de/cyface/deserializer/UnzippedPhoneDataDeserializer.java

```diff
@@ -267,8 +267,7 @@ private MetaData queryForMetaData(final Connection connection, final long measur
         final var lengthResultSet = lengthQuery.executeQuery();
         lengthResultSet.next();
         final var length = lengthResultSet.getDouble(1);
-        final var version = "1";
-        return new MetaData(measurementIdentifier, deviceType, osVersion, appVersion, length, userId, version);
+        return new MetaData(measurementIdentifier, deviceType, osVersion, appVersion, length, userId, MetaData.CURRENT_VERSION);
```

Comment:
Data which is deserialized by the current code is in the current deserialized version (which is "2.0.0"), i.e. this was a typo.